### ISSUE TITLE
pt-verify-checksum should mention when it skips files

### ISSUE THREAD

**Issue by mbanck** [uptimejp/postgres-toolkit#46]
`pt-verify-checksum` claims it has "verified X files", even if that file is skipped by `check_filename`. That is very misleading, it should print an `INFO` in verbose mode and mention it verified 0 files if it only skipped files and did not actually ran `verifychecksum.bin` on any of them.

**Comment by mbanck** [comment]
Patch for this:

```
--- pt-verify-checksum.orig     2018-01-23 14:17:55.000000000 +0100
+++ pt-verify-checksum  2018-01-23 14:27:29.000000000 +0100
@@ -105,8 +105,10 @@
                 log.debug("verifing %s" % f)
                 if self.verify_one(f) is False:
                     corrupted = corrupted + 1
-            
-            count = count + 1
+                count = count + 1
+            else:
+                if self.verbose is True:
+                    log.info("Skipped %s" % f)
 
         log.info("Verified %d files. %d files corrupted." % (count, corrupted))
 
```

**Comment by snaga** [comment]
Committed at f89a468.

**Comment by snaga** [comment]
Merged #53.
